### PR TITLE
Security and messaging overhaul,  rebranding

### PR DESCRIPTION
**Pull request by jaytmiller**
Most substantive changes are in src/index.ts.
Corresponding changes also exist in the server and CLI announce script not included here.
Corresponding changes are needed in the science-platform-images repo to build the new labextension.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 BSD 3-Clause License
 
 Copyright (c) 2020, Josh Geden All rights reserved.
+Copyrigth (c) 2024, Space Telescope Science Institute, All rights reserved.
```
Comment (msanchezst) [inline LICENSE:4]: minor typo here

File: package.json [hunk 2]
```diff
@@ -1,86 +1,87 @@
...
+        "jupyterlab-extension"
+    ],
+    "homepage": "https://github.com/spacetelescope/stsci-announce",
+    "bugs": {
+        "url": "https://github.com/spacetelescope/stsci-announce/issues"
+    },
+    "license": "BSD-3-Clause",
+    "author": {
+        "name": "Josh Geden",
+        "email": "jgeden@lbl.gov"
```
Comment (msanchezst) [inline package.json:17]: Do we need to add more authors here?

File: setup-jh-dev [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+set -eux
+
+if [[ $# != 1 ]]; then
+    echo "usage: $0  <branch-to-checkout>"
+    exit 1
+else
+    BRANCH=$1
+fi
+
+git clone https://github.com/jaytmiller/stsci-announce.git
```
Comment (msanchezst) [inline setup-jh-dev:16]: do we want to use your fork for this?

**Review by msanchezst — APPROVED** [review]
Looks great! Some minor comments

File: package.json (hunk 2 above)
Comment (jaytmiller) [reply package.json:17]: That's a good point.   I don't think you can,  but we could add a maintainers section I believe and leave Josh as-is:   https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors.  The repo itself is already tracking contributors.

File: LICENSE (hunk 1 above)
Comment (jaytmiller) [reply LICENSE]: Got it,  thanks!

File: setup-jh-dev (hunk 3 above)
Comment (jaytmiller) [reply setup-jh-dev]: This never worked right really and I think the entire dev process may also be changing;  decided to drop it altogether,  have to figure it out again next time anyway.

**Pull request merged**